data/eks_dockerimage-replication/versions/1.22.yaml

```diff
@@ -0,0 +1,10 @@
+ami:
+  version: 1.22.17-20230217
```

Comment:
Is there a need to support 1.22 going forward?

Reply:
There is no support from eks for any versions under 1.23 ATM. So deleting them